src/components/Auth/UserAccountInfo.scss

```
@@ -79,18 +79,42 @@
   }
 }
 
-// Full account view
+// Full account view.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)